linode/manager
# removed: [M3-7554, M3-7559] - Post Release Cleanup Parent/Child Feature Flags

## Description 📝
Remove the `parentChildAccountAccess` feature flag logic now that the feature has been in production for some time.

## Changes  🔄
- Removed feature flags from code, unit tests, and cypress e2e tests

## Target release date 🗓️
Next release: 5/28

## How to test 🧪

### Prerequisites
### Verification steps
- Run unit tests: `yarn test`
- Run E2E tests:
```
yarn cy:run -s "cypress/e2e/core/account/account-cancellation.spec.ts,cypress/e2e/core/account/account-login-history.spec.ts,cypress/e2e/core/account/display-settings.spec.ts,cypress/e2e/core/account/personal-access-tokens.spec.ts,cypress/e2e/core/account/user-permissions.spec.ts,cypress/e2e/core/account/user-verification-banner.spec.ts,cypress/e2e/core/account/users-landing-page.spec.ts,cypress/e2e/core/billing/billing-contact.spec.ts,cypress/e2e/core/billing/restricted-user-billing.spec.ts,cypress/e2e/core/parentChild/account-switching.spec.ts,cypress/e2e/core/parentChild/token-expiration.spec.ts"
```
- Ensure that parent child features are not displayed for default accounts.
- Ensure that the features are enabled for parent/child accounts.

## As an Author I have considered 🤔

*Check all that apply*


---
## Commit message and pull request title format standards

> **Note**: Remove this section before opening the pull request
**Make sure your PR title and commit message on squash and merge are as shown below**

`<commit type>: [JIRA-ticket-number] - <description>`

**Commit Types:**
- `feat`: New feature for the user (not a part of the code, or ci, ...).
- `fix`: Bugfix for the user (not a fix to build something, ...).
- `change`: Modifying an existing visual UI instance. Such as a component or a feature.
- `refactor`: Restructuring existing code without changing its external behavior or visual UI. Typically to improve readability, maintainability, and performance.
- `test`: New tests or changes to existing tests. Does not change the production code.
- `upcoming`: A new feature that is in progress, not visible to users yet, and usually behind a feature flag.

**Example:** `feat: [M3-1234] - Allow user to view their login history`

---
